src/components/Patient/PatientDetailsTab/ImmunisationRecords.tsx

```diff
@@ -43,7 +43,7 @@ export const ImmunisationRecords = (props: PatientProps) => {
           <div className="flex flex-row gap-x-4">
             <h1 className="text-xl">{t("covid_details")}</h1>
             <button
-              className="hidden rounded border border-secondary-400 bg-white px-1 py-1 text-sm font-semibold text-green-800 hover:bg-secondary-200 group-hover:flex"
+              className=" rounded border border-secondary-400 bg-white px-1 py-1 text-sm font-semibold text-green-800 hover:bg-secondary-200 group-hover:flex"
               disabled={!patientData.is_active}
```

Comment:
Let's switch to using shadcn's button instead to maintain consistent design easily.

Reply:
@rithviknishad Should i change the existing button to shadcns' . If so should i replace other areas with the button component as well ?

Reply:
Yup! Let's do that

Reply:
Do update the title too